react-main/src/app/components/TopbarProfile/index.js

```diff
@@ -0,0 +1,18 @@
+import React, { Component, Fragment } from 'react';
+import styles from './styles.scss';
+import logout from '../../../images/singout.svg';
+import toPlay from '../../../images/toPlay.svg';
+
+class TopbarProfile extends Component {
```

Comment:
The idea is to have the components share the same TopBar, not to have one for each screen the app has. This would repeat code and add a TopBar component for each screen. You should have something of this sorts:

```
function MainComponent() {
 return (
   <TopBar someProps />
   <MainApp /> // component that has the routes
 )
}

```

And the top bar should render the user name or the logout button depending on if the user is logged in